conformance/common/src/main/kotlin/com/connectrpc/conformance/ssl/SSL.kt

```diff
@@ -55,96 +58,3 @@ fun sslContext(): Pair<SSLSocketFactory, X509TrustManager> {
 
     return result.sslSocketFactory() to result.trustManager
 }
-
-// https://github.com/connectrpc/conformance/blob/main/cert/client.crt
-// cert issues: https://stackoverflow.com/questions/9210514/unable-to-find-valid-certification-path-to-requested-target-error-even-after-c
-private const val CLIENT_CERT = """-----BEGIN CERTIFICATE-----
```

Comment:
Changed to just read these in as resources instead of including them as string constants here. This will make it easier to update if we ever need to do so again.

Reply:
Good call.